src/Speckle.Sdk/Serialisation/SerializationResult.cs

```diff
@@ -2,13 +2,25 @@
 
 public readonly record struct SerializationResult(Json Json, Id? Id);
 
-public readonly record struct Json(string Value)
+public readonly record struct Json
 {
+  public Json(string json)
+  {
+    Value = json ?? throw new ArgumentNullException(nameof(json));
```

Comment:
My only comment is, useually if you're performing a null check, then you should do 
`public Json([NotNull] string? json)`

That way, you never force consumers to do an additional unnecessary null check, and if `null` is truly expected, then you're typing it correctly.

Reply:
It's really not expected.  I'm being extra defensive because something at run time is ignoring the nullability then causing the error being seen.  I don't want people to think it's okay.